docs/spec/spec.md

```diff
@@ -151,8 +151,12 @@ The conforming implementation of the specification is released and included in t
         * 12.3.2 [Client](#1232-client)
             * 12.3.2.1 [SSL/TLS](#12321-ssltls)
             * 12.3.2.2 [Mutual SSL](#12322-mutual-ssl)
-13. [Tools](#13-tools)
-    * 13.1 [GraphiQL Client](#131-graphiql-client)
+13. [Federated Subgraph](#13-federated-subgraph)
+    * 13.1 [The `@graphql:Subgraph` Annotation](#131-the-graphqlsubgraph-annotation)
+    * 13.2 [The `@graphql:Entity` Annotation](#132-the-graphqlentity-annotation)
+    * 13.2 [The `ReferenceResolver`](#133-the-referenceresolver)
+14. [Tools](#13-tools)
+    * 14.1 [GraphiQL Client](#131-graphiql-client)
```

Comment:
We might have to update the links in the BBEs

Reply:
PR: https://github.com/ballerina-platform/ballerina-distribution/pull/4161